src/deluge/io/midi/device_specific/midi_device_lumi_keys.cpp

```diff
@@ -0,0 +1,267 @@
+/*
+ * Copyright (c) 2023 Aria Burrell (litui)
```

Comment:
I'm not opposed to this but it's new for the project - we should discuss at the meeting on sunday

Reply:
That can be changed if needed. I figured on altogether new files it made sense, and GPL continues to apply regardless.

Reply:
Yeah I agree, I guess I'm wondering if we should check in with other people who contributed brand new files

Reply:
Making a note to bring this up Sunday if I attend but otherwise I'm ready to merge.